### ISSUE TITLE
Not possible to disable a rule for part of the code

### ISSUE THREAD

**Issue by mgol** [eslint/eslint#871]
Provided the following code:

``` js
var a;
/* jshint bitwise: false */
/* eslint no-bitwise: 0 */
a = a & a;
/* eslint no-bitwise: 2 */
/* jshint bitwise: true */
```

jsHint passes it and ESLint fails. I suspect ESLint is not disabling the rule only for part of the file but that the latter comment overwrites the first and that's what's applied to the whole file.

Tested with version from master.
